### ISSUE TITLE
[Bug]: Mistakenly caused an infinite crash loop in the Ainsel River Well Depths Site of Grace

### ISSUE THREAD

**Issue by mjs24shanahan** [LukeYui/EldenRingSeamlessCoopRelease#383]
### Describe the issue in detail

Me and my two other tarnished buddies encountered a strange bug where I as the host could not enter or get close to any underground area without a crash.  We thought we would be super smart and have the other tarnished host me in his world and then travel to the underground SoG, I was a wanderer in his world.  I loaded into the SoG and then instantly my game crashed.  Now in my save file every time I load in the game crashes.  Please help!

### Steps to recreate the bug

1. Don't backup save files like you should b/c you know this is a beta and there will be some bugs.
2. Have problems loading into any underground area only with modded save file.
3. Be very arrogant and think you can solve issues on your own.
4. Ask tarnished buddy to host instead and since the underground does not crash for them, load former host in as wanderer to tarnished buddy's host world and then instantly proceed to crash.
5. Restart Modded Game, use continue or load game with the intention of hosting and then instantly crash.
6. Repeat step 5 until upset.

### Version

1.2.5 (Latest)

**Comment by Lilmega123** [comment]
I encountered something very similar while playing with my buds. I was underground going through the hidden tunnel at Volcano Manor. As I went down my game crashed and kept doing so every time I tried to go toward the manors underground area. At some point I got bored of trying so I teleported away (grace site was "Schoolhouse Classroom" at the Academy of Raya Lucaria) and since then my game has entered into an infinite crash loop. Every time I load into the game with my character (that character specifically) the game crashes immediately.

**Comment by foopfoodoo** [comment]
Same issue happened with me. I was the host and I was accessing Siofra River for the first time but I kept crashing on my way down the elevator. The other person I was playing with did not have this problem. When they became the host and I the guest and tried it that way I was still crashing. Only problem now is that whenever I load my game I load into Siofra River and immediately crash so now I got an infinite crash loop.
